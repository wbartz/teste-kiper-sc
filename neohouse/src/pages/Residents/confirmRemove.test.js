import { shallow } from 'enzyme';
import React from 'react';
import ConfirmRemove from './confirmRemove';

describe('ConfirmRemove', () => {
  let wrapper;
  const props = {
    history: {
      go: jest.fn(),
    },
    onClose: jest.fn(),
    user: {},
    show: false,
  };

  beforeEach(() => {
    wrapper = shallow(<ConfirmRemove {...props} />);
  });

  it('render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
