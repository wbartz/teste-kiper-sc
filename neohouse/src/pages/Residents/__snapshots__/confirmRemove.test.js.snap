// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmRemove render without crashing 1`] = `ShallowWrapper {}`;
